### PR TITLE
STSMACOM-694 - Pop-up Note > Display MetaSection component

### PR DESCRIPTION
**Pull request by Terala-Priyanka**
## Purpose
Issue: [STSMACOM-694](https://issues.folio.org/browse/STSMACOM-694)

## Screenshots
image

image

### CODE REVIEW

File: lib/Notes/NotePopupModal/NotePopupModal.js [hunk 1]
```diff
@@ -161,6 +162,13 @@ const NotePopupModal = ({
 
   const hideDelete = !stripes.hasPerm('ui-notes.item.delete');
 
+  const metaSection = (
```
Comment (NikitaSedyx) [inline lib/Notes/NotePopupModal/NotePopupModal.js:165]: I believe it's better to not show ViewMetaData when note or metadata are undefined

Comment (Terala-Priyanka) [reply lib/Notes/NotePopupModal/NotePopupModal.js]: @NikitaSedyx 
Implemented the change. Please review

File: lib/Notes/NotePopupModal/NotePopupModal.js [hunk 2]
```diff
@@ -161,6 +162,12 @@ const NotePopupModal = ({
 
   const hideDelete = !stripes.hasPerm('ui-notes.item.delete');
 
+  const metaSection = popupNote?.metadata && (
+    <ViewMetaData
+      metadata={popupNote?.metadata}
```
Comment (NikitaSedyx) [inline lib/Notes/NotePopupModal/NotePopupModal.js:167]: ?. is not required now at this place

**Review by NikitaSedyx — APPROVED** [review]
lgtm, just a nit

**Review by JohnC-80 — APPROVED** [review]

**Pull request merged**